packages/scripts/src/Case.ts

```diff
@@ -0,0 +1,76 @@
+import z from "zod";
+
+// Map of embedding model name -> vector (array of numbers)
+export const Embeddings = z.record(z.string(), z.number().array());
+
+export type Embeddings = z.infer<typeof Embeddings>;
+
+export const PromptAndEmbeddings = z.object({
+  prompt: z.string(),
+  embeddings: Embeddings,
+});
+
+export type PromptAndEmbeddings = z.infer<typeof PromptAndEmbeddings>;
+
+export const RelevanceMetrics = z.object({
+  // normalized square magnitude difference (lower = closer = better)
+  norm_sq_mag_diff: z.number(),
+  // cosine similarity (are vectors pointing the same way?) [-1, 1]
+  cos_similarity: z.number(),
```

Comment:
can you include in docs comments on here/somewhere what these metrics actually measure? is it how specific a prompt is to its answer? what's the difference between the 2 metrics as well?